opts/envdir.go

```diff
@@ -0,0 +1,43 @@
+package opts
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+)
+
+/*
+Read in a directory with files acting as environment variables.
+*/
+func ParseEnvDir(directory string) ([]string, error) {
+	dir, err := ioutil.ReadDir(directory)
+	if err != nil {
+		return []string{}, err
+	}
+
+	lines := []string{}
+	for _, info := range dir {
+		// ignore directories
+		if !info.IsDir() {
+			variable := info.Name()
+			if strings.ContainsAny(variable, whiteSpaces) {
```

Comment:
Also worth noting that when I originally wrote `ParseEnvDir`, I skipped "hidden" files, that began with `.`, but opted out of it because `envdir` does not skip them. Maybe there's valid reason to name a var `.FOO`? I've never done it, but I don't see rejecting the idea.

And to my previous point, we don't disallow this inside of `-e .FOO=bar` or in an `env-file`.